File: C/search_insert_position.c
int searchInsert(int* nums, int numsSize, int target){
    int index = 0;
        
    for(int i = 0; i < numsSize; i++){
        if(target > nums[i]){
            index = i + 1;
            continue;
        }
        if(target == nums[i]){
            index = i;
            continue;
        }
    }    
    return index;
}

File: Python3/search_insert_position.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        index = 0;
        
        for i in range(len(nums)):
            if target > nums[i]: 
                index = i+1
                continue
            if target == nums[i]: 
                index = i
                continue
                
        return index
